Microsoft/WebTemplateStudio
# App Service - create an instance

This PR handles the creation of an App Service web app.

To test:
- Launch the extension in preview mode
- Take note of the project name
- Add App Service on the Azure page (do "Create a new resource group")
- Create your project (try with different backend framework)
- Go to Azure portal (https://ms.portal.azure.com/#home)
    - Go to the Resource Group page and go into <project name>_<timestamp> resource group
    - Make sure that there is an ASP and a web app in the resource group
    - Go into the web app, then "Application Settings (Classic)", check that the Runetime stack corresponds to the backend framework 

Note: It is possible to create an instance in an existing resource group, just make sure that there is no Functions App in that resource group because we cannot have different kind of ASP in the same resource group.
